### PR TITLE
Use incoming grid area from request instead of creating a new grid ar…

### PR DESCRIPTION
**Pull request by erduna**
## Description
Currently we create a grid area every time we create an MP. The grid area created uses a guid instead of a 3 digit code. This PR uses the incoming grid area from the request as the associated grid area for the MP when creating it, and does not create a new separate grid area every time.

When creating a metering point we do not create an associated grid area, but check that it is correctly tied to an existing one through our validations. 

Finally, creation and maintenance of grid areas is out of scope, so the implementation of a grid area will be done at a future date. 

## References

* #126

### CODE REVIEW

File: source/Energinet.DataHub.MeteringPoints.Domain/MeteringPoints/MeteringPoint.cs [hunk 1]
```diff
@@ -24,7 +24,7 @@ public abstract class MeteringPoint : AggregateRootBase
         protected ProductType _productType;
         #pragma warning restore
         private Address _address;
-        private GridAreaId _gridAreaId;
+        private string _gridAreaId;
```
Comment (cmgrd) [inline source/Energinet.DataHub.MeteringPoints.Domain/MeteringPoints/MeteringPoint.cs:27]: Why the change to string?

**Review by cmgrd — CHANGES_REQUESTED** [review]

**Review by kft — CHANGES_REQUESTED** [review]
`GridArea` is concept in the metering point domain. Metering points are connected/associated with a `GridArea`, but it is a separate entity type which the `MeteringPoint` entity should not be concerned with. Thus, `MeteringPoint` should just hold a reference to the identifier of `GridArea` and not the name or code of grid area. We have just not implemented the `GridArea` concept yet.

What is the point of replacing the id with the code/name ?

**Comment by erduna** [comment]
Closing because approach is wrong (as is our current approach)
